src/App.css

```diff
@@ -1,10 +1,11 @@
+/* stylelint-disable no-descending-specificity */
 html {
   box-sizing: border-box;
   font-size: 62.5%;
 }
 
 body {
```

Comment:
- [ ] [Optional] The Styling should be EXACTLY the same as the one provided on Zeplin, kindly adjust the sizes, padding/margin so your design is identical to the given one 

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220728_201818](https://user-images.githubusercontent.com/25227022/181620327-33f0f007-8b0d-4a7e-b6f8-3a0cc0216f1a.png)|![Screenshot_20220728_202309](https://user-images.githubusercontent.com/25227022/181620617-526044ba-030d-4bf4-a1b8-c09cce1e341e.png)|